app/lib/methods/urlParse.ts

```diff
@@ -0,0 +1,3 @@
+import * as uri from 'uri-js';
+
+export const urlParse = (url: string): string => uri.serialize(uri.parse(url));
```

Comment:
I'm not sure about this.
We already have `parseUrls` https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/lib/methods/helpers/parseUrls.ts
Having `urlParse` is going to create confusion for sure.

Also I'm not sure how useful it is to export this code
```jsx
uri.serialize(uri.parse(url))
```

instead of the actual reusable thing

```jsx
let newUrl = url.replace(/\/+$/, '').replace(/\\/g, '/');
const ascii = /^[ -~\t\n\r]+$/;
if (isIOS && !ascii.test(newUrl)) {
	newUrl = urlParse(newUrl);
}
```

Maybe we could increase the scope and unify some URL logic, as we also have `parseQuery`?
Ping me in chat :)